### ISSUE TITLE
Windows 10

### ISSUE THREAD

**Issue by Stuwilli1** [phw198/OutlookGoogleCalendarSync#852]
*Are you already using OGCS?**
Provide the version number.
I am a current user for 2.8.1

**Describe the question you have**
A clear and concise description of your question and what you would like clarifying.

I have received a new Windows laptop (Windows 10) and trying to install OGSC.  After downloading the setup and program files, then hitting the setup, attached window pops up wanting to add a new account and I never get the PGSC to fully open.  Office 365 is already installed and active, is that the issue?

![ogsc](https://user-images.githubusercontent.com/39503028/65718036-af172780-e070-11e9-979b-324e0b6ac607.jpg)

please advise.  Thank you.

**Comment by phw198** [comment]
What do you mean by installed and "active"? From your screenshot, it doesn't look like you've configured Outlook yet, so OGCS has no Outlook profile to connect to. Before trying to use OGCS, have you opened Outlook and set up a calendar yet?

**Comment by Stuwilli1** [comment]
Yes two calendars have been active and running so no idea why it is doing this 

Stuart

**Comment by phw198** [comment]
:spiral_notepad: Please provide your `OGcalsync.log` file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).

**Comment by Stuwilli1** [comment]
Paul, the program never gets to open so this file is not installed I am
presuming?  Search the appdata file and nothing.  It is very odd and my
last laptop had no issue installing

Best regards,

Stuart

**Comment by phw198** [comment]
OGCS is already installed and running, else you wouldn't get the "Application terminated window". 

Are you saying that the folder `%APPDATA%\Outlook Google Calendar Sync` doesn't exist at all? What about `%LOCALAPPDATA%\OutlookGoogleCalendarSync`?
